be/src/geo/geo_types.cpp

```diff
@@ -31,6 +34,18 @@
 
 namespace doris {
 
+GeoPoint::GeoPoint() : _point(new S2Point()) {}
```

Comment:
You can go directly  implement these in geo_type.h simple construction destructors